introduction.md

```diff
@@ -8,3 +8,8 @@ This repository contains a list of dojo rules for the Deadly Vipers dojo
 
 +I prefer git path :)
 +I prefer javascript path :)
+
+* git
+* emacs
+* csharp
+* perl
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?